scene/gui/graph_edit.cpp

```diff
@@ -1926,6 +2249,55 @@ bool GraphEdit::is_valid_connection_type(int p_type, int p_with_type) const {
 	return valid_connection_types.has(ct);
 }
 
+void GraphEdit::attach_graph_element_to_frame(const StringName &p_graph_element, const StringName &p_parent_frame) {
+	GraphFrame *frame = Object::cast_to<GraphFrame>(get_node(NodePath(p_parent_frame)));
+	ERR_FAIL_NULL_MSG(frame, "Parent frame does not exist or is not a GraphFrame.");
+	linked_parent_map.insert(p_graph_element, p_parent_frame);
+	frame_attached_nodes[p_parent_frame].insert(p_graph_element);
+
+	_update_graph_frame(frame);
+
+	_ensure_node_order_from_root(p_graph_element);
+}
+
+void GraphEdit::detach_graph_element_from_frame(const StringName &p_graph_element) {
+	if (!linked_parent_map.has(p_graph_element)) {
+		return;
+	}
+
+	GraphFrame *frame = Object::cast_to<GraphFrame>(get_node(NodePath(linked_parent_map[p_graph_element])));
+
+	ERR_FAIL_NULL_MSG(frame, "Parent frame does not exist or is not a GraphFrame.");
+
+	frame_attached_nodes.get(frame->get_name()).erase(p_graph_element);
+	linked_parent_map.erase(p_graph_element);
+
+	_update_graph_frame(frame);
+}
+
+GraphFrame *GraphEdit::get_frame(const StringName &p_attached_graph_element) {
+	if (!linked_parent_map.has(p_attached_graph_element)) {
+		return nullptr;
+	}
+
+	Node *parent = get_node_or_null(NodePath(linked_parent_map[p_attached_graph_element]));
+
+	return Object::cast_to<GraphFrame>(parent);
+}
+
+TypedArray<StringName> GraphEdit::get_attached_noded_of_frame(const StringName &p_graph_frame) {
```

Comment:
typo: `get_attached_noded_of_frame` -> `get_attached_nodes_of_frame`

Reply:
I also haven't exposed this method by accident. Fixed.